src/components/FindMovie/FindMovie.tsx

```diff
@@ -16,42 +62,56 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className="input"
+              value={query}
+              onChange={(event) => setQuery(event.target.value)}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {movie === false && query === prevQuery.current && (
```

Comment:
```suggestion
          {!movie && query === prevQuery.current && (
```

Reply:
I need here **movie === false**, because movie can be null
**const [movie, setMovie] = useState<Movie | null | false>(null);**